recipes/conversation_assistant/README.md

```diff
@@ -1,7 +1,7 @@
 # Conversation
 > Chat with TJBot!
 
-This recipe uses the [Watson Assistant](https://www.ibm.com/watson/services/conversation/), [Speech to Text](https://www.ibm.com/watson/services/speech-to-text/), and [Text to Speech](https://www.ibm.com/watson/services/text-to-speech/) services to turn TJ into a chatting robot.
+This recipe uses the [watsonx Assistant](https://www.ibm.com/watson/services/conversation/), [Speech to Text](https://www.ibm.com/watson/services/speech-to-text/), and [Text to Speech](https://www.ibm.com/watson/services/text-to-speech/) services to turn TJ into a chatting robot.
```

Comment:
New watsonx Assistant link: https://www.ibm.com/products/watsonx-assistant

Reply:
Oy, the STT/TTS links don't work anymore either. Oh but I see below you've updated them. So we just need to update them here too. :)